content/plugins/lektor-markdown-ruby-blocks/lektor-markdown-highlighter/contents.lr

```diff
@@ -0,0 +1,13 @@
+_model: plugin
```

Comment:
This file is in the wrong location.

It needs to be in `content/plugins/lektor-markdown-ruby-blocks/contents.lr`.

Right now it's one level too deep in `content/plugins/lektor-markdown-ruby-blocks/lektor-markdown-highlighter/contents.lr`. As such, Lektor does not find and build the page.